CV.html

```diff
@@ -0,0 +1,169 @@
+
+      <ul>
+        <li><strong>2018-2021</strong></li>
+      </ul>
+      <p class="cv-items"><job>Facultatea de Economie si Administrarea Afacerilor </job>Iasi - Romania</p>
+    </div>
+    <div>
+      <h2 class="blue-color">Skills</h2>
+      <div class="skills">
+        <ul>
+          <li class="skills-item">
+            <strong>Tech Skills</strong>
+            <tech>HTML5</tech>
+            <tech>CSS3</tech>
+            <tech>Visual Studio Code</tech>
+            <tech>Microsoft</tech>
+            <tech>AutoCAD</tech>
+          </li>
+        </ul>
+        <ul>
+          <li class="skills-item">
+            <strong>Soft Skills</strong>
+
+            <tech>Curious</tech>
+            <tech>Agile</tech>
+            <tech>Motivated</tech>
+            <tech>Teamwork</tech>
+            <tech>Networking</tech>
+          </li>
+        </ul>
+        <ul>
+          <li class="skills-item">
+            <strong>Additional Skills</strong>
+            <tech>Analytical Skills</tech>
+            <tech>Organizational Skills</tech>
+            <tech>Atention to detail</tech>
+            <tech>Language Proficiency</tech>
+            <tech>Presentation Skills</tech>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <div>
+      <h2 class="blue-color">Leadership and Volunteering Experiences</h2>
+      <ul>
+        <li><strong>Voluntary - Community Support Foundation</strong></li>
+        <div class="skills-list">
+          <li>- Supporting disadvantaged community groups through medical, social and educational programs;</li>
+          <li>- Supporting the development of local communities;</li>
+        </div>
+      </ul>
+      <ul>
+        <li><strong>Quality Analyst</strong></li>
+        <div class="skills-list">
+          <li>- Responsible for the implementation of the quality management system;</li>
+          <li>- Represents and liaises between the company and the beneficiary regarding issues related to quality control;</li>
+          <li>- Exercises the quality control of works by execution phases;</li>
+          <li>- Responsible for the implementation of the quality management system;</li>
+        </div>
+      </ul>
+      <ul>
+        <li><strong>Junior Accountant</strong></li>
+        <div class="skills-list">
+          <li>- Filing and registration of supporting documents;</li>
+          <li>- Verification of sales and purchase logs;</li>
+          <li>- Checking the cash register and the bank register;</li>
+          <li>- Recording monthly attendance and generating payroll</li>
+        </div>
+      </ul>
+    </div>
+    <div>
+      <h2 class="blue-color">Achievements</h2>
+      <ul>
+        <li>Driver License Cat.B</li>
+        <li>Make-Up Artist Certification</li>
+        <li>Economist Degree</li>
+      </ul>
+    </div>
+
+    <div class="form">
+    <h2 class="blue-color">Create an account</h2>
+    <form id="emailForm">
```

Comment:
(optional) - in html nu se prea practica folosirea stilului de camelCase, ci mai mult kebab-case. In acest caz ar fi `id=email-form`